### ISSUE TITLE
Move Draco extension to 2.0 subdirectory

### ISSUE THREAD

**Issue by pjcozzi** [KhronosGroup/glTF#1246]
Sorry, I should have caught this in #874, but the new directory:
```
glTF/extensions/Khronos/KHR_draco_mesh_compression
```
Should really be in the new-ish 2.0 subdirectory
```
extensions/2.0/Khronos
```

No rush I suppose but it will break fewer links.

CC @outofcontrol

**Comment by pjcozzi** [comment]
The link in https://github.com/KhronosGroup/glTF/blob/master/extensions/README.md and any other deep links should, of course, also be updated.

**Comment by pjcozzi** [comment]
Fixed in #1247, thanks @donmccurdy!
